content

```diff
@@ -0,0 +1 @@
+website/generated-content
```

Comment:
What is this file?

Reply:
Not needed on master. Deleted it from the PR.
Needed on the asf-site branch as a symlink from content to website/generated-content and already added in that branch.